src/metaswitch/ellis/remote/homestead.py

```diff
@@ -268,9 +268,9 @@ def _http_request(url, callback, overload_retries=3, **kwargs):
     retries_holder = {'retries': overload_retries}
 
     def do_http_request():
-        _log.debug("Sending HTTP %s request to %s",
-                   kwargs.get('method', 'GET'),
-                   url)
+        _log.info("Sending HTTP %s request to %s",
```

Comment:
Incoming requests get INFO-level logs (e.g. `02-06-2016 17:02:47.842 UTC INFO web.py:1447: 200 DELETE /accounts/live.tests@example.com/numbers/sip%3A6505550094%40cw-ngv.com (0.0.0.0) 58.43ms`). It makes sense for outgoing requests to appear at the same level - and practically, it would be nice to get them without turning debug logging on.